test/integration/targets/setup_mariadb/tasks/setup_mariadb.yml

```diff
@@ -0,0 +1,64 @@
+# We run two servers listening different ports
+# to be able to check replication (one server for master, another for standby).
+
+- name: Install MariaDB packages on RedHat family OS
+  yum:
+    name: "{{ item }}"
+    enablerepo: epel
+  loop:
+  - mariadb-server
+  - mariadb
+  when: ansible_os_family == 'RedHat'
+
+- name: Create directories for standby
+  file:
+    state: directory
+    path: "{{ item }}"
+    owner: mysql
+    group: mysql
+  loop:
+  - "{{ standby_datadir }}"
+  - "{{ standby_logdir }}"
+
+- name: Copy cnf template
+  template:
+    src: my.cnf.j2
+    dest: /etc/my.cnf
+    owner: mysql
+    group: mysql
+    force: yes
+
+- name: Initialize standby's DB
+  shell: 'mysql_install_db --user=mysql --datadir={{ standby_datadir }}'
+
+- name: Initialize master's DB
+  shell: 'mysql_install_db --user=mysql --datadir={{ default_datadir }}'
+
+- name: Start services
+  shell: 'mysqld_multi start 1,2'
+
+- name: Pause
+  pause: seconds=3
+
+########### For painful debug uncomment the lines below ##
```

Comment:
This is great and will save people time in the future, thanks.

Reply:
i was suffering for 3 days trying to launch 2 MariaDB instances in one container and configure replication between them :) so decided to left these lines :)